apps/preview/nuxt/pages/showcases/Combobox/ComboboxBasic.vue

```diff
@@ -162,10 +162,10 @@ const handleReset = () => {
 };
 
 const handleBlur = () => {
-  isValid.value = !!options.find((option) => option.value.toLowerCase() === inputModel.value.toLowerCase());
+  isValid.value = !!countriesList.find((option) => option.value.toLowerCase() === inputModel.value.toLowerCase());
```

Comment:
question: from where those `countriesList` coming from I can't see it defined in component

Reply:
It's below

Reply:
ah okay, my bad